student.rb

```diff
@@ -1,9 +1,17 @@
 require_relative 'person'
 # Student class
 class Student < Person
+  attr_reader :classroom
+
   def initialize(classroom, age, name = 'Unknown', parent_permission: true)
```

Comment:
[OPTIONAL]
- [ ] kindly make sure the user will be able to put another value on `parent_permission` other than `true`

YOu may achieve it by saying:
```suggestion
  def initialize(classroom, age, name = 'Unknown', parent_permission = true)
```

Reply:
@christianbiring1 Okay, I appreciate your feedback. However, in accordance with the `rubocop` best practice rule, it is advised to use a key-value pair for the Boolean value of the optional arguments